Assets/Scripts/Networking/PlayerManager.cs

```diff
@@ -207,14 +248,12 @@ public void OnPhotonSerializeView(PhotonStream stream, PhotonMessageInfo info)
             if (stream.IsWriting)
             {
                 // We own this player: send the others our data
-                stream.SendNext(this.IsFiring);
```

Comment:
Why don't we have to send this data?

Reply:
The shooting is client side, we will use RPCs to tell other players that the local player is shooting. I will explain in a meeting, just remind me